### PR TITLE
add arkon chain

### PR DESCRIPTION
**Pull request by NLGRF**
- add arkon chain
- add arkon logo

### CODE REVIEW

File: _data/chains/eip155-1678.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "symbol": "KON",
+    "decimals": 18
+  },
+  "infoURL": "https://arkon.network/",
+  "shortName": "kon",
+  "chainId": 1678,
+  "networkId": 1678,
+  "explorers": [{
+    "name": "Arkon Explorer",
+    "url": "https://testnet.arkscan.net/",
```
Comment (ligi) [inline _data/chains/eip155-1678.json:20]:
```suggestion
    "url": "https://testnet.arkscan.net",
```

**Review by ligi — CHANGES_REQUESTED** [review]
I cannot resolve QmdZYBAwkZMV2UqE91oqEkNsd6pjP7bSjz3GR7N24nw7Lj - where did you pin it?

**Comment by NLGRF** [comment]
> I cannot resolve QmdZYBAwkZMV2UqE91oqEkNsd6pjP7bSjz3GR7N24nw7Lj - where did you pin it?

I upload logo to pinata.cloud. @ligi

**Comment by NLGRF** [comment]
Please review again. @ligi

**Comment by NLGRF** [comment]
Please review again. @ligi @bmann

**Pull request closed without merging**